src/Core/Components/List/FluentCombobox.razor

```diff
@@ -2,12 +2,11 @@
 @inherits ListComponentBase<TOption>
 @typeparam TOption
 
-@if (!String.IsNullOrEmpty(Height))
-{
-    <style>
-        @($"#{Id}::part(listbox) {{ max-height: {Height}; z-index: {ZIndex.ComboboxPopup} }}")
-    </style>
-}
+
+<style>
+    @($"#{Id}::part(listbox) {{ max-height: {(string.IsNullOrEmpty(Height) ? null : Height)}; z-index: {ZIndex.ComboboxPopup} }}")
```

Comment:
Could you modify the generation of this "string", because when "height" is not defined, the style still contains a "height" attribute.

Example
```html
<style>#combobox::part(listbox) { max-height: ; z-index: 9995 }</style>
```

Reply:
Sure. I'll amend the changes

Reply:
Before:
![Screenshot 2023-12-22 093152](https://github.com/microsoft/fluentui-blazor/assets/49217428/cbfaee66-2c6c-4874-8c99-0c5d2900172a)

After:
![Screenshot 2023-12-22 093405](https://github.com/microsoft/fluentui-blazor/assets/49217428/356dbfd0-42f7-45d7-8074-ef4881d95155)

Reply:
I'm going to be annoying but it's my job 😎. Can you add a space after the max-height? To be consistent with the rest of the generated code. 

And it's probably clearer to do ´$"max-height: {Height};"´.

Reply:
No prob at all